### PR TITLE
Fixed Typo

### PR DESCRIPTION
**Pull request by karanankit01**

### CODE REVIEW

File: src/SourceWriter.h [hunk 1]
```diff
@@ -13,13 +13,13 @@ class SourceWriter {
     int lines_ = 1;
     bool volatile_ = false;
     char lastChar_;
-    int intiailColumn_ = 0;
+    int initalColumn_ = 0;
```
Comment (Koihik) [inline src/SourceWriter.h]: initial

Comment (karanankit01) [reply src/SourceWriter.h]: opps ..

**Comment by tammela** [comment]
Hi Ankit,

make sure you compile your code before opening a PR.

**Comment by tammela** [comment]
Hi @karanankit01,

please squash all commits into one.

**Comment by karanankit01** [comment]
Hi @tammela 
Thanks for guiding, I will keep these things in mind from now .

**Pull request merged**